packages/pages/src/dev/server/server.ts

```diff
@@ -61,22 +62,41 @@ export const createServer = async (
     ));
   }
 
-  const loadedFunctions = await loadServerlessFunctions();
-  const loadedFunctionPaths = [...loadedFunctions.keys()].map(
-    (slug) => "/" + slug
-  );
+  // Load functions from their source files
+  const functionModules: Map<string, FunctionModuleInternal> = new Map();
```

Comment:
Can't you use FunctionModuleCollection here?

Reply:
I know the comments for FMC was that it's indexed by name, but do we actually use it that way? If it's always by slug then update the comment.